### ISSUE TITLE
Process multiple font formats

### ISSUE THREAD

**Issue by s4msung** [battlesnake/gulp-google-webfonts#46]
Optionally, when specifying a list of formats, they all should be downloaded and css with all formats should be generated

**Comment by Detzler** [comment]
yeah, it should be possible to download all format types. woff2, woff, eot, etc.
